db/model/GdocXTag.ts

```diff
@@ -0,0 +1,8 @@
+import { Entity, BaseEntity, PrimaryColumn } from "typeorm"
+
+@Entity("posts_gdocs_x_tags")
+export class GdocXTag extends BaseEntity {
+    static table = "posts_gdocs_x_tags"
```

Comment:
I don't know what this is for

Reply:
It appears to be a convention to use when making a Knex query (e.g. `await db.knexTable(Gdoc.table)`) so I figured I'd just put it everywhere